test_quick.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+import pytest
+from random import shuffle
+from quick import quick
+
+
+@pytest.fixture(scope="function")
+def presorted():
+    presorted = range(10)
+    return presorted
+
+
+@pytest.fixture(scope="function")
+def unsorted():
+    unsorted = [8, 4, 6, 3, 5, 1, 9, -3, 10, 2, 5.5, 7]
+    return unsorted
+
+
+@pytest.fixture(scope="function")
+def binary():
+    binary = [1, 0, 0, 0, 1, 1, 0, 1, 1, 1, 0, 0]
+    return binary
+
+
+@pytest.fixture(scope="function")
+def one():
+    one = [5]
+    return one
+
+
+@pytest.fixture(scope="function")
+def empty():
+    empty = []
+    return empty
+
+
+@pytest.fixture(scope="function")
+def big():
+    big = range(10000)
+    shuffle(big)
+    return big
```

Comment:
I actually didn't know about Shuffle. 
Another way to do this same thing is ```random.sample(range(10), 10)```
The nice thing about that is you can bump up the range so you could have ```random.sample(range(1000000), 10)``` or 10 random numbers out of million.
Just a fun fact. Your way totally works!